tess_area.cpp

```diff
@@ -0,0 +1,85 @@
+#define CSV_IO_NO_THREAD
+#include "csv.h"
+
+#include <cmath>
+#include <cstdlib>
+
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <numeric>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+int main() {
+  std::vector<double> radius;
```

Comment:
Declare variable when you are going to use them, unlike `c` you don't have to define all the variables up front.